src-vue/src/i18n/lang/ru.json

```diff
@@ -74,7 +74,7 @@
             "by": "от",
             "remove": "Удалить мод",
             "remove_dialog_title": "Внимание",
-            "remove_dialog_text": "Удалить мод Thunderstore?",
+            "remove_dialog_text": "Удалить мод?",
```

Comment:
I think the `Thunderstore` mention should stay. Already reverted the change on Weblate, just waiting for it to propagate to GitHub ^^